src/app/services/map-cell.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@angular/core';
+import { Response } from '@angular/http';
+import { Point } from 'geojson';
+import { Observable } from 'rxjs/Observable';
+
+import { MapCell } from '../models/map-cell.model';
+import { LabApiHttp } from '../auth/api-http.service';
+import { apiHost } from '../constants';
+
+@Injectable()
+export class MapCellService {
+
+    constructor(private apiHttp: LabApiHttp) {}
+
+    nearest(point: Point, distance: number): Observable<MapCell[]> {
+        const url = `${apiHost}/api/map-cell/${point.coordinates[1]}/${point.coordinates[0]}?distance=${distance}`;
```

Comment:
This leaves off the trailing slash which is causing a redirect to occur when this accesses the API. It all still works, but it'd be good to include the trailing slash to avoid the extra roundtrip for the redirect response.